src/main/java/uk/gov/pay/directdebit/common/model/subtype/ServiceUserName.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.pay.directdebit.common.model.subtype;
+
+import java.util.Objects;
+
+public class ServiceUserName {
+
+    private final String serviceUserName;
+
+    private ServiceUserName(String serviceUserName) {
```

Comment:
As discussed in another comment, this is a SUN name rather than a “service user name”.

So I think this can just be `SunName` or something similar.

And since this is a class, you could stick some Javadoc on it so people are in no doubt as to what it is:

```java
/**
 * The name associated with the Service User Number (SUN) registered with Bacs
 * For Direct Debit payments, this is what appears on the bank statement
 */
```